### DISCUSSION TITLE
wi-fi and zapret

### DISCUSSION THREAD

**Discussion by ThinkForge-core** [bol-van/zapret#930] [Q&A] [answered]
Приветствую. Настраивал Запрет по кабелю с ПК на Кинетике, который включен в основной роутер. Только для ютюба. На ПК по кабелю всё заработало. При подключении к Кинетику по wi-fi с планшета - на планшете не работает. Вот тут уже нет понимания - должно ли работать по беспроводной сети при таком подходе или нужна другая настройка. Help

**Comment by AlexDiag** [comment]
Планшет наверное Android ? Если да- так там приложение Ютуба получает видео иначе чем браузер на компьютере. WiFi тут видимо ни при чем. Стратегия обхода иная нужна

**Reply by ThinkForge-core** [reply, marked as answer]
Да планшет на Android. Только на нём я использую не приложение  Ютуба, а браузеры. И как оказалось всё намного проще. Нужно было заставить браузер открывать версию ютуб для ПК, т.е уйти от этой m.youtube.com к youtube.com, что получилось не сразу, а только отключив zapret. После всё заработало. WiFi как таковой действительно ни причем.
